package.json

```diff
@@ -83,7 +83,8 @@
     "build:local-packages": "yarn workspace @upswyng/types build && yarn workspace @upswyng/common build",
     "build:server": "yarn build:local-packages && yarn workspace @upswyng/server build",
     "build:web": "yarn build:local-packages && yarn workspace @upswyng/web build",
-    "build:local-db": "docker compose up --build -d && mongorestore --port 27018 --username upswyng-dev-user --password upswyng123 -d upswyng-dev ./packages/server/localdb/mongodump",
+    "build:local-db": "docker-compose up --build -d && npm run restore:db",
+    "restore:db": "mongorestore --port 27018 --username upswyng-dev-user --password upswyng123 -d upswyng-dev ./packages/server/localdb/mongodump",
```

Comment:
the `build:local-db` command is kinda broken since it doesn't wait to run the `restore` command
broke this out as a half step towards fixing -- proper would be to create a script that waits for mongodb to be ready to accept connections and and then run

in progres

Reply:
You are definitely right! I don't think we have ever had any issues with it requesting to restore before mongo was ready, but it's definitely a possibility and if it did it for you then we should get this fixed 🚀

Reply:
We could add the "wait for mongo to be ready and then restore" part to this script https://github.com/CodeForBoulder/upswyng/blob/master/start_local.sh that we have recently created so that new devs don't have to struggle to stand the project up?

Reply:
👍